### PR TITLE
Feature/list controller v2 new unit tests

### PR DESCRIPTION
**Pull request by spaceo**
Adding unit tests for ListType and ListItem classes

## How to review
Look at commit
Run tests

### CODE REVIEW

File: tests/phpunit/Unit/ListItemTest.php [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+use App\Enums\ListType;
+use Symfony\Component\HttpKernel\Exception\UnprocessableEntityHttpException;
+
+class ListItemTest extends TestCase
+{
+    public function testThatItCanCreateAListItemFromACorrectUrlParameter()
+    {
+        $listItem = ListItem::createFromUrlParameter('999-base:id');
+        $this->assertInstanceOf(ListItem::class, $listItem);
+        $this->assertCount(4, (array)$listItem);
```
Comment (xendk) [inline tests/phpunit/Unit/ListItemTest.php]: I'd `assertEquals()` and check if the array matches the expected. This implicitly checks the count and also tests that it actually returns the expected and not just an array of four empty strings.

**Review by xendk — APPROVED** [review]
